src/test/kotlin/uk/gov/justice/digital/hmpps/managerecallsapi/component/pact/provider/ManageRecallsUIPactTest.kt

```diff
@@ -106,7 +104,7 @@ class ManagerRecallsUiAuthorizedPactTest : ManagerRecallsUiPactTestBase() {
 
   @State("a recall exists")
   fun `a recall exists`() {
-    recallRepository.save(recallWithPopulatedFields(::RecallId.zeroes(), nomsNumber, emptySet()))
+    recallRepository.save(fullyPopulatedRecall(::RecallId.zeroes()).copy(documents = emptySet()))
```

Comment:
Why does documents need to be empty?  (Perhaps the state needs a more descriptive name now)?